gherkin-utils/java/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.cucumber</groupId>
+    <artifactId>cucumber-parent</artifactId>
+    <version>2.1.1</version>
+  </parent>
+
+  <artifactId>gherkin-utils</artifactId>
+  <version>2.0.1-SNAPSHOT</version>
+
+  <name>gherkin-utils</name>
+  <url>https://github.com/cucumber/cucumber/tree/master/gherkin-utils</url>
+
+  <properties>
+    <project.Automatic-Module-Name>io.cucumber.gherkin.utils</project.Automatic-Module-Name>
+  </properties>
+
+  <scm>
+    <connection>scm:git:git://github.com/cucumber/cucumber.git</connection>
+    <developerConnection>scm:git:git@github.com:cucumber/cucumber.git</developerConnection>
+    <url>git://github.com/cucumber/cucumber.git</url>
+  </scm>
+
+  <dependencies>
+      <dependency>
+          <groupId>io.cucumber</groupId>
+          <artifactId>messages</artifactId>
+          <version>13.1.0</version>
+      </dependency>
+      <dependency>
+          <groupId>org.junit.jupiter</groupId>
+          <artifactId>junit-jupiter</artifactId>
+          <version>5.7.0</version>
+          <scope>test</scope>
+      </dependency>
+    <dependency>
+      <groupId>io.cucumber</groupId>
+      <artifactId>gherkin</artifactId>
+      <version>15.0.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Let your IDE autoformat this file.

Reply:
Sr, this is my mistake. I have already fixed.